src/main/fc/rc.c

```diff
@@ -132,6 +133,11 @@ float getRcDeflectionAbs(int axis)
     return rcDeflectionAbs[axis];
 }
 
+float getMaxRcDeflectionAbs(void)
+{
```

Comment:
```suggestion
float getMaxRcDeflectionAbs(void) {
```

https://betaflight.com/docs/development/CodingStyle#opening-braces